dev/integration_tests/ios_add2app/ios_add2appTests/IntegrationTests.m

```diff
@@ -112,8 +88,8 @@ - (void)testDualFlutterView {
         (DualFlutterViewController *)navController.visibleViewController;
     GREYAssertNotNil(viewController,
                      @"Expected non-nil DualFlutterViewController.");
-    waitForFlutterSemanticsTree(viewController.topFlutterViewController);
-    waitForFlutterSemanticsTree(viewController.bottomFlutterViewController);
+    [self expectSemanticsNotification:viewController.topFlutterViewController];
+    [self expectSemanticsNotification:viewController.topFlutterViewController];
```

Comment:
Did you mean bottomFlutterViewController?

Reply:
Yes - oops.